src/LayerPlan.h

```diff
@@ -577,8 +581,12 @@ class LayerPlan : public NoCopy
      * \param wall_0_wipe_dist The distance to travel along the wall after it has been laid down, in order to wipe the start and end of the wall together
      * \param flow_ratio The ratio with which to multiply the extrusion amount
      * \param always_retract Whether to force a retraction when moving to the start of the wall (used for outer walls)
+     * \param unretract_before_last_travel_move Whether to unretract before the last travel move of the travel path,
```

Comment:
Architecturally it seems a bit weird for the `addWall` function to get this parameter.

Should the `unretract_before_last_travel_move` parameter of paths not just be set by whatever code adds the travel move that the unretract is necessary for?

Reply:
Indeed, but the codes that add travel moves do not know if they are adding outer or inner wall to make a decision on the unretraction. The knowledge of whether we are adding an outer wall lies in the code that actually adds that wall.

Reply:
But that code does know whether to make that small travel move towards the position away from the outer wall, right? How does it decide on that then, if it doesn't know whether it's printing an outer or an inner wall?

Reply:
As discussed, this is now fixed, since the decision on whether we should unretract is taken by the combing (considering that this is the place where the extra travel move to the inside of an outer wall is added).